client/package-lock.json

```diff
@@ -0,0 +1,2063 @@
+{
```

Comment:
We use `yarn.lock` everywhere else, unless there is some reason for not using yarn here?